api/types/types.go

```diff
@@ -75,6 +75,22 @@ type Image struct {
 	Labels      map[string]string
 }
 
+type ImageJSON struct {
```

Comment:
Just asking, since daemon.Repositories().Images(conf) is returning types.Image, can't you just update that struct and refactor when it needs to get refactored?